### ISSUE TITLE
stdout comes back as null

### ISSUE THREAD

**Issue by mgenev** [Medium/local-dynamo#27]
```
node bin/launch_local_dynamo.js --database_dir=/Users/mgenev1271/Projects/temp/local-dynamo/db --port=4567
```

```
/Users/mgenev1271/Projects/temp/local-dynamo/bin/launch_local_dynamo.js:15
childProcess.stdout.pipe(process.stdout)
                   ^
TypeError: Cannot read property 'pipe' of null
    at Object.<anonymous> (/Users/mgenev1271/Projects/temp/local-dynamo/bin/launch_local_dynamo.js:15:20)
    at Module._compile (module.js:430:26)
    at Object.Module._extensions..js (module.js:448:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:471:10)
    at startup (node.js:117:18)
    at node.js:948:3
```

**Comment by themucha** [comment]
I submitted a pull request that fixes this issue at https://github.com/Medium/local-dynamo/pull/26.

**Comment by mgenev** [comment]
tested the PR and it works

**Comment by shichongrui** [comment]
Is this going to get fixed? Seems the PR #26 has been closed without merging or any explanation. I am still encountering this issue.
